### PR TITLE
Vars to enums

### PR DESCRIPTION
**Pull request by nev-r**
shiny

### CODE REVIEW

**Comment by nev-r** [comment]
next up:
bucket enums?

File: src/app/item-popup/ItemStat.tsx [hunk 1]
```diff
@@ -14,13 +14,14 @@ import { DestinySocketCategoryStyle } from 'bungie-api-ts/destiny2';
 import { getSocketsWithStyle } from '../utils/socket-utils';
 import PressTip from 'app/dim-ui/PressTip';
 import { getPossiblyIncorrectStats } from 'app/utils/item-utils';
-import { RECOIL_DIRECTION, TOTAL_STAT_HASH } from 'app/search/d2-known-values';
+import { TOTAL_STAT_HASH } from 'app/search/d2-known-values';
+import { ItemCategoryHashes, StatHashes } from 'data/d2/generated-enums';
 
 // used in displaying the modded segments on item stats
 const modItemCategoryHashes = [
   1052191496, // weapon mods
```
Comment (delphiactual) [inline src/app/item-popup/ItemStat.tsx:22]:
```suggestion
  ItemCategoryHashes.WeaponModsDamage, // weapon mods
```

Comment (nev-r) [reply src/app/item-popup/ItemStat.tsx:22]: tyyyyyyyy

**Comment by nev-r** [comment]
i tried to delete redundant ones and leave a few "why this" ones but i will look it over again

File: src/app/wishlists/wishlists.ts [hunk 2]
```diff
@@ -87,8 +80,8 @@ function isWeaponOrArmorOrGhostMod(plug: DimPlug): boolean {
   if (
     plug.plugItem.itemCategoryHashes?.find(
       (ich) =>
-        ich === INTRINSIC_ITEM_CATEGORY || // intrinsics
-        ich === WEAPON_GAMEPLAY_MODS_CATEGORY || // weapon gameplay socket
+        ich === ItemCategoryHashes.WeaponModsIntrinsic ||
+        ich === ItemCategoryHashes.WeaponModsGameplay ||
         ich === 3851138800 // armor gameplay socket
```
Comment (delphiactual) [inline src/app/wishlists/wishlists.ts:85]: this is not in `generated-enums.ts`

Comment (nev-r) [reply src/app/wishlists/wishlists.ts:85]: yep, see https://github.com/DestinyItemManager/DIM/issues/5521

**Pull request merged**